release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1108,7 +1108,7 @@ They correspond to older, similarly named SCCs, but with the following enhanceme
 * `seccompProfile` is defaulted to `runtime/default` if unset. In previous releases, this field had to be empty.
 * `allowPrivilegeEscalation` must be unset or set to `false` in security contexts. Previously, a `true` value was allowed.
 
-New clusters allow use of the `restricted-v2` SCC for any authenticated user in place of the `restricted` SCC; the `restricted` SCC is no longer available to users of new clusters, unless the access is explicitly granted. Clusters that are upgraded to {product-title} 4.11 from previous versions allow both `restricted-v2` and `restricted` SCCs.
+In {product-title} 4.11, the `restricted-v2` SCC is now the default SCC for new installations, instead of the `restricted` SCC. New clusters allow use of the `restricted-v2` SCC for any authenticated user in place of the `restricted` SCC; the `restricted` SCC is no longer available to users of new clusters, unless the access is explicitly granted. Clusters originally installed in {product-title} 4.10 or earlier will continue allowing all authenticated users to use the `restricted` SCC when upgrading to {product-title} 4.11 and later. This keeps the default security permissions for {product-title} as secure-by-default and aligns with pod security admission and pod security standards from the upstream Kubernetes project, while keeping the permissions of upgraded clusters consistent with their previous state.
```

Comment:
This is a long sentence; consider breaking it into shorter sentences for clarity. The clause starting from semicolon onwards could be another sentence.